amazon-eks-nodegroup.yaml

```diff
@@ -217,7 +210,7 @@ Resources:
       Description: Allow node to communicate with each other
       GroupId: !Ref NodeSecurityGroup
       SourceSecurityGroupId: !Ref NodeSecurityGroup
-      IpProtocol: '-1'
+      IpProtocol: -1
```

Comment:
Is it really ok to remove the quotes here? `IpProtocol` takes a string argument, and `-` should be quoted since it's an indicator character.

Reply:
This should be enclosed in quotes